compiler/lookups.nim

```diff
@@ -255,9 +255,10 @@ proc errorUseQualifier*(c: PContext; info: TLineInfo; s: PSym) =
   var candidate = initIdentIter(ti, c.importTable.symbols, s.name)
   var i = 0
   while candidate != nil:
-    if i == 0: err.add " --use "
-    else: err.add " or "
-    err.add candidate.owner.name.s & "." & candidate.name.s
+    if i == 0: err.add " --use one of the following:\n"
```

Comment:
While you are at it, could you add space after the `--`? It's meant as a long-dash, which typically has symmetric whitespace. My brain was parsing `--use` as a command line switch all the time ;).

Reply:
@bluenote10 done, thanks. I've updated the 'expected output' above.